lib/middleware.js

```diff
@@ -34,6 +34,20 @@ module.exports.IntegrationMiddlewareChain = function IntegrationMiddlewareChain(
   };
 };
 
+module.exports.DestinationMiddlewareChain = function DestinationMiddlewareChain() {
+  var apply = middlewareChain(this);
+
+  this.applyMiddlewares = function(facade, integration, callback) {
+    return apply(
+      function(mw, payload, next) {
+        mw({ payload, integration, next });
```

Comment:
In modern JS you can do `mw({ payload, integration, next });`. Unfortunately AJS is retro JS, so this needs to be 
```
  mw({
          payload: payload,
          integration: integration,
          next: next,
      });
```